src/mudlet.cpp

```diff
@@ -266,6 +266,7 @@ mudlet::mudlet()
     menuHelp->setToolTipsVisible(true);
     menuAbout->setToolTipsVisible(true);
 
+    // FIXME: add a command line option to change this?
```

Comment:
Maybe open a public issue here instead of adding TODO comments to source?

Reply:
Initially a coding note to myself - remember I use the **ToDo** plugin so it is helpful to *me* - but will make that issue before I merge this PR and link to it from this comment.

Reply:
Issue created #4807 .